tools/jdk/BUILD

```diff
@@ -195,6 +195,26 @@ java_import(
     neverlink = 1,
 )
 
+java_import(
+    name = "ImportDepsChecker",
+    jars = ["//tools/jdk:ImportDepsChecker_deploy.jar"]
+)
+
+java_binary(
```

Comment:
Technically, this is used only in Android code, but it looks like a useful Java tool, so move on.